upstream/update.py

```diff
@@ -197,8 +199,8 @@ def download(source: str, release: Release) -> Release:
     manifest = SOURCES[source]["manifest"]
     dest = FILEDIR / source / "manifests" / release.name / manifest
     dest.parent.mkdir(exist_ok=True)
-    urllib.request.urlretrieve(release.path, dest)
-    return Release(release.name, str(dest), release.size)
+    urllib.request.urlretrieve(release.upstream, dest)
+    return Release(release.name, dest, release.size)
```

Comment:
it looks like something broke down and needed repair in this script?

Reply:
```python
>>> thisisfine = Path('https://ohno.com/bazinga')
>>> try:
...     urllib.request.urlretrieve(thisisfine)
... except Exception as e:
...     print('OH NO! %s' % e)
...     print('you went for {}'.format(thisisfine))
...     print(f'but see? Path ate your slashy bit!')
... 
OH NO! expected string or bytes-like object
you went for https:/ohno.com/bazinga
but see? Path ate your slashy bit!
>>> 
```